dev/dev/devDeployment.yaml

```diff
@@ -42,6 +42,10 @@ datarepo-api:
     OPENCENCUS_SPRING_ENABLED: false
     GOOGLE_SECURE_FOLDER_RESOURCE_ID: "753276429356"
     USERMETRICS_BARDBASEPATH: https://terra-bard-dev.appspot.com
+    OIDC_CLIENTID: bbd07d43-01cb-4b69-8fd0-5746d9a5c9fe
+    OIDC_AUTHORITYENDPOINT: https://terradevb2c.b2clogin.com/terradevb2c.onmicrosoft.com/B2C_1A_SIGNUP_SIGNIN/v2.0
+    OIDC_ADDCLIENTIDTOSCOPE: true
+    OIDC_EXTRAAUTHPARAMS: prompt=login
   serviceAccount:
```

Comment:
Does this file also need the added secret key for existingSecretNameOauth and 
existingOauthClientSecretSecretKey?

Reply:
Nice eye!  I'm glad you noticed.  It actually doesn't since B2C doesn't require a secret, so we just leave it off (I wanted an example of the B2C config as well) to show that that looks like.  Eventually it will all look like this but while it's not approved for prod, we need the other envs to point to google